### PR TITLE
delete unwanted domains that i don't need rn & update information

### PR DESCRIPTION
**Pull request by xydv**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [ ] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
You don't own any of these domains, so you cannot modify them.

**Comment by dedomil** [comment]
that @adityaasyadav account is me myself only!

**Comment by WilliamDavidHarrison** [comment]
You will need to comment on this PR with that account to confirm that.

**Comment by adityaasyadav** [comment]
hey @dedomil & @adityaasyadav both accounts are mine!

**Comment by dedomil** [comment]
please look into it, i commented from my alt account

**Review by wdhdev — APPROVED** [review]

File: domains/l.aditya.json
Comment (WilliamDavidHarrison) [inline domains/l.aditya.json:1]: Each part of your subdomain name must be 2 characters or more.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Pull request merged**